test/suites/greenboot.robot

```diff
@@ -0,0 +1,109 @@
+*** Settings ***
+Documentation       Tests related to greenboot
+Resource            ../resources/common.resource
+Resource            ../resources/systemd.resource
+Resource            ../resources/ostree-health.resource
+Resource            ../resources/microshift-config.resource
+Library    SSHLibrary
+Suite Setup         Setup Suite 
+Suite Teardown      Teardown Suite
+
+
+*** Variables ***
+${LOCAL_CONFIG_FILE}                  ../docs/config/busybox_running_check.sh
+${REMOTE_CONFIG_FILE}                 /etc/greenboot/check/required.d/50_busybox_running_check.sh
+${GREENBOOT_CONFIG_FILE}              /etc/greenboot/greenboot.conf
+${GREENBOOT_CONFIG_CONTENT}           MICROSHIFT_WAIT_TIMEOUT_SEC=180
+${MANIFEST_SOURCE_DIR}                ./assets/kustomizations/greenboot/
+${MANIFEST_DIR}                       /etc/microshift/manifests
+${MICROSHIFT_CONFIG}                  /etc/microshift/config.yaml
+*** Test Cases ***
+
+# user-workload-health
+
+GreenbootHealth Run with User Workload
+    [Documentation]    check
```

Comment:
I think this test is verifying that the workload starts and then greenboot is successful?